### DISCUSSION TITLE
VSR Quality 5

### DISCUSSION THREAD

**Discussion by essi2** [emoose/DLSSTweaks#135] [General]
I notice in the tooltip it says

"Nvidia Control Panel can let you pick from quality 1 - 4, but the video setting itself allows choosing up to quality 5 - it's unknown whether this gives any image quality improvement (power draw seems the same as quality 4 at least)."

From what I can tell switching around the qualities in NV Control panel and refreshing the Tweaker, Quality 5 seems almost certainly to be what NV Control Panel calls 'Auto'.

**Comment by emoose** [comment]
Yep that text was added early when VSR first launched, there was only 1 - 4 back then but I'd noticed driver did expose option 5, then later they released auto mode which uses that option.
(no idea if the auto mode released in later drivers is the same what option 5 did before that mode released though)
Anyway thanks for reminding me to change that text, will look into it when I add the DLSS 3.7 updates.
